packages/x-scheduler/src/joy/event-calendar/useEventCalendar.test.ts

```diff
@@ -0,0 +1,127 @@
+import { renderHook, act } from '@mui/internal-test-utils';
+import { DateTime } from 'luxon';
+import { spy } from 'sinon';
+import { useEventCalendar } from './useEventCalendar';
+import { getAdapter } from '../../primitives/utils/adapter/getAdapter';
+
+const DEFAULT_PARAMS = { events: [] };
+
+describe('useDateNavigation', () => {
```

Comment:
Tests adapted from `useDateNavigation`

I'll add tests for the other methods later on